src/customizations/components/theme/Header/Header.jsx

```diff
@@ -10,7 +10,6 @@ import { connect } from 'react-redux';
 import eeaFlag from '../../../../../theme/themes/eea/assets/images/Header/eea.png';
 import logo from '../../../../../theme/themes/eea/assets/images/Header/eea-logo.svg';
 import searchIcon from '../../../../../theme/themes/eea/assets/images/Header/search.png';
```

Comment:
Only some components (those that are Volto overrides) need to stay in src/customizations. Everything else doesn't belong here. Place images closer to the components so that this sort of long paths are avoided. If not possible otherwise, make a webpack alias for the theme assets.